### PR TITLE
Add Babel to server

### LINKED ISSUE #33: Add Babel support on server
Add Babel on the server code so that ES6 modules and more features can be used.

### PR DESCRIPTION
**Pull request by prakhar1912**
Babel support has been added in the server folder. Now ES6 modules and features can be used. resolves #33

### CODE REVIEW

**Comment by niinpatel** [comment]
Change your commit message to match this: https://udacity.github.io/git-styleguide/

I missed it in your previous pull request but its already been merged so we can't do anything about that one.

**Comment by prakhar1912** [comment]
> Change your commit message to match this: https://udacity.github.io/git-styleguide/
> 
> I missed it in your previous pull request but its already been merged so we can't do anything about that one.

@niinpatel Babel support would fall under `chore` ?

**Comment by niinpatel** [comment]
> > Change your commit message to match this: https://udacity.github.io/git-styleguide/
> > I missed it in your previous pull request but its already been merged so we can't do anything about that one.
> 
> @niinpatel Babel support would fall under `chore` ?

I think yes!

File: server/__tests__/index.test.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
-const request = require('supertest');
-const server = require('../src/index');
+
+import '@babel/polyfill';
```
Comment (niinpatel) [inline server/__tests__/index.test.js]: You could just specify useBuiltIns: 'usage' in .babelrc
image

Comment (prakhar1912) [reply server/__tests__/index.test.js]: Good catch. Thanks!

**Comment by niinpatel** [comment]
Other than that. LGTM.

**Pull request merged**